### PR TITLE
Sflow on mgmt VRF support

### PR DESCRIPTION
**Pull request by venkatmahalingam**
Sflow support on mgmt VRF - this enables the sflow to reach the collector via mgmt VRF.

Signed-off-by: Venkatesan Mahalingam <venkatesan_mahalinga@dell.com>

### CODE REVIEW

File: doc/mgmt/SONiC_Management_Framework_sFlow.md [hunk 1]
```diff
@@ -192,7 +193,25 @@ Syntax:
 sonic(config)# no sflow collector 1.1.1.1
 sonic(config)#
 ```
+##### Add sFlow Collector with VRF name
+Synxtax:
```
Comment (jeffyin-dell) [inline doc/mgmt/SONiC_Management_Framework_sFlow.md]: Spelling for "Syntax"

**Review by jeffyin-dell — APPROVED** [review]

**Comment by GarrickHe** [comment]
@venkatmahalingam  - Should we make it explicit that we only support default and mgmt VRF right now?

File: doc/mgmt/SONiC_Management_Framework_sFlow.md (hunk 1 above)
Comment (venkatmahalingam) [reply doc/mgmt/SONiC_Management_Framework_sFlow.md]: will fix it.

**Comment by venkatmahalingam** [comment]
> @venkatmahalingam - Should we make it explicit that we only support default and mgmt VRF right now?

Yes, will mention that explicitly.

**Pull request merged**